### ISSUE TITLE
Default Audio Output to Speaker

### ISSUE THREAD

**Issue by ahmadmuhsink** [Gruveo/gruveo-ios-sdk#5]
Gruveo version : reproduced in 1.0-1.1

Problem:
When establishing video call without earphone.
I expect the audio output to use loudspeaker (bottom) not the call speaker phone (top).
Tapping on the speaker icon in the gruveo call screen can fix this, but the interaction is bugged.
the first 2 taps is doing nothing. speaker icon still at `on` state without any effect, third tap will output the sound to correct loudspeaker. after that tapping on speaker icon behave correctly (toggle on/off).

in short:
1. speaker icon in calling screen
2. audio output

Current solution:
In our app on `callEstablished` delegate I set the `AVAudioSession ` mode to video chat to make sure audio output to loudspeaker.
something like this:
```
AVAudioSession *audioSession = [AVAudioSession sharedInstance];
[audioSession setMode:AVAudioSessionModeVideoChat error:nil];
```

Question:
1. is this expected behaviour? because in the previous screen in our app, our code access `AVAudioSession` as well and that may cause above bug.
2. how about handle this within gruveo itself? set `AVAudioSession` mode based on `videoChat` paremeter `YES`/`NO`

**Comment by artmatsak** [comment]
The SDK should be routing sound to the loudspeaker by default when establishing a video call. We're looking into this.

**Comment by artmatsak** [comment]
Fixed in the latest version of the iOS SDK (1.1.2).
